sky/templates/gcp-ray.yml.j2

```diff
@@ -43,6 +43,12 @@ available_node_types:
     node_config:
       labels:
         skypilot-user: {{ user }}
+      {%- if num_reserved_nodes > 0 %}
```

Comment:
```suggestion
      {%- if specific_reservations > 0 %}
```

Reply:
Should it be:
```suggestion
      {%- if num_reserved_workers > 0 %}
```
?

Reply:
oops, I was trying to say 
```suggestion
      {%- if specific_reservations %}
```

we cannot use `num_reserved_workers` as that only represents the workers that need to be reserved — if we are launching a single node, that variable will always be 0.